includes/class-wc-payments-webhook-processing-service.php

```diff
@@ -524,7 +524,6 @@ private function process_webhook_dispute_created( $event_body ) {
 		$event_type   = $this->read_webhook_property( $event_body, 'type' );
```

Comment:
`nit` Feel free to ignore.

Maybe not needed but I happen to notice `$event_type` is not used in this function, so we might be able to delete it.

Reply:
Agreed – resolved in dcae68f.